### PR TITLE
fs-2511-remove-default-fund-round

### PR DESCRIPTION
**Pull request by gio-karunakaran**
Since COF R2W3 has closed and fs2505 is complete (passing fund and round information through the flow of authentication and frontend) we can now remove the default fund and rounds used in authenticator and call the specified fund and round passed to authenticator from frontend as query parameters.

### CODE REVIEW

**Review by ramsharma-prog — APPROVED** [review]

File: models/fund.py [hunk 1]
```diff
@@ -62,5 +53,7 @@ def get_service_name():
         if short_name:
             fund_data = FundMethods.get_fund(fund_short_name=short_name)
         else:
-            fund_data = FundMethods.get_fund(fund_id=Config.DEFAULT_FUND_ID)
+            fund_data = FundMethods.get_fund(
+                fund_id=Config.DEFAULT_FUND_ID, use_short_name=False
```
Comment (srh-sloan) [inline models/fund.py:57]: get_fund doesn't take an arg called fund_id?
Do we have a unit test for this method?

File: models/fund.py [hunk 2]
```diff
@@ -35,24 +35,15 @@ def add_round(self, fund_round: Round):
...
-            url = (
-                Config.FUND_STORE_API_HOST + Config.FUND_STORE_FUND_ENDPOINT
-            ).format(fund_id=fund_short_name)
-        # TODO remove after R2W3 closes and fs-2505 is complete (make fund_short_name non-optional) # noqa
-        else:
-            url = (
-                Config.FUND_STORE_API_HOST + Config.FUND_STORE_FUND_ENDPOINT
-            ).format(fund_id=fund_id)
-
+    def get_fund(fund_short_name: str, use_short_name: bool = True) -> Fund:
```
Comment (srh-sloan) [inline models/fund.py:38]: if the only arg is fund_short_name, why do we need the bool as well?

**Review by srh-sloan — COMMENTED** [review]
A few comments around the get_fund method but otherwise looks good 👍

File: models/fund.py (hunk 1 above)
Comment (gio-karunakaran) [reply models/fund.py:57]: ahh you're right, i recon we can remove that whole else block entirely 👍🏾

File: models/fund.py (hunk 2 above)
Comment (gio-karunakaran) [reply models/fund.py:38]: I kept this is in as a future proofing thing (incase we do ever want to pass around ids aswell as short_names) but on second thought we can add this in the future fairly easily if needed so will remove! 👌🏾

File: models/fund.py (hunk 1 above)
Comment (gio-karunakaran) [reply models/fund.py:57]: we do have a test but it looks outdated, let me fix this to!

**Review by ramsharma-prog — APPROVED** [review]
LGTM

**Pull request merged**